File: app/src/main/java/com/artamonov/currencyconverter/main/mapper/RatesMapper.java
package com.artamonov.currencyconverter.main.mapper;

import com.artamonov.currencyconverter.main.networking.models.CurrencyRateJson;
import com.artamonov.currencyconverter.main.networking.models.Rate;
import java.util.List;

public interface RatesMapper {

    List<Rate> map(final CurrencyRateJson rateItemsJson);
}

File: app/src/main/java/com/artamonov/currencyconverter/main/mapper/RatesMapper.kt
package com.artamonov.currencyconverter.main.mapper

import com.artamonov.currencyconverter.main.networking.models.CurrencyRateJson
import com.artamonov.currencyconverter.main.networking.models.Rate

interface RatesMapper {
    fun map(rateItemsJson: CurrencyRateJson?): MutableList<Rate>
}